vector/src/main/java/im/vector/activity/VectorHomeActivity.java

```diff
@@ -940,6 +945,7 @@ private void updateSelectedFragment(final TabLayout.Tab item, boolean isAnimated
                     fragment = TchapContactFragment.newInstance();
                 }
                 mCurrentFragmentTag = TAG_FRAGMENT_PEOPLE;
+                resetFilter();
```

Comment:
We want to keep the search pattern when we switch between "Conversations" and "Contacts" tabs
We could not reset filter here

Reply:
If we keep the search pattern when we switch from the conversation to contact tab, nothing is displayed for the contacts. For his contacts to appear, the user must think about deleting his search; which is not very user friendly.

I think it's better to clear the search on tab switch because it's not a global search, but a specific search for each tab (conversation vs contacts).

Reply:
Ok, I will test the new code now that the bug concerning the search has been fixed.